README.md

```diff
@@ -131,39 +131,39 @@ If both build-info.plist and build-info.yaml are present, the plist file will be
 
 ####build-info keys
 
-**distribution_style**  
+**distribution_style**
```

Comment:
The extra spaces in the realm cause a line break. If you look at the file after your edits you'll see the text now follows the term without a line break.